CRM/Export/Form/Select.php

```diff
@@ -94,23 +94,16 @@ public function preProcess() {
     $this->_componentIds = array();
     $this->_componentClause = NULL;
 
-    $stateMachine = $this->controller->getStateMachine();
-    $formName = CRM_Utils_System::getClassName($stateMachine);
-    $isStandalone = $formName == 'CRM_Export_StateMachine_Standalone';
+    $this::$queryMode = $this->controller->get('component_mode');
```

Comment:
Can we go with a get function rather than relying on this being set at the right time ie

function getQueryMode()....

Reply:
@eileenmcnaughton Agreed - much nicer.  I've updated the PR